AUTHORS

```diff
@@ -123,6 +128,8 @@ Thomas Backlund <tmb@mageia.org>
 Topi Miettinen <toiwoton@gmail.com>
 Vasiliy Tolstov <v.tolstov@selfip.ru>
 Wim Muskee <wimmuskee@gmail.com>
+dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
```

Comment:
This shouldn't really be here, right?

Reply:
Removed dependabot based on feedback from @pvalena .